server/graphql/v2/object/Individual.js

```diff
@@ -102,7 +102,9 @@ export const GraphQLIndividual = new GraphQLObjectType({
         type: GraphQLBoolean,
         async resolve(collective, args, req) {
           const user = await req.loaders.User.byCollectiveId.load(collective.id);
-          return twoFactorAuthLib.userHasTwoFactorAuthEnabled(user);
+          if (req.remoteUser.id === user.id) {
```

Comment:
There is no guarantee that `req.remoteUser` is set right?

Reply:
All the queries where we fetch this info are logged in, but you're right - there's no guarantee of it. I copied the snippet from V1's `hasTwoFactorAuth`, I'll fix it there and in `hasPassword` that uses the same logic too.